package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "description": "openaps oref0 reference implementation of the reference design",
   "scripts": {
-    "test": "make test"
+    "test": "mocha tests/*.js"
```

Comment:
@LorelaiL what was your reason for changing from make to mocha?  Was make (the way we had it configured) just not running all of the tests?